src/main/java/Duke.java

```diff
@@ -1,10 +1,140 @@
+                System.out.println("\t " + list.get(taskNo));
+                System.out.println("\t Now you have " + (amount - 1) + " tasks in the list.");
+                System.out.println(lineBreak);
+                list.set(taskNo, "deleted");
+                save(list);
+            } else if (response.matches("todo .+")) {
+                System.out.println(lineBreak);
+                System.out.println("\t Got it. I've added this task:");
+                System.out.println("\t   [T][ ] " + response.substring(5));
+                System.out.println("\t Now you have " + (amount + 1) + " tasks in the list.");
+                System.out.println(lineBreak);
+                list.add(amount, "[T][ ] " + response.substring(5));
+                amount++;
+                save(list);
+            } else if (response.matches("deadline .+")) {
+                System.out.println(lineBreak);
+                System.out.println("\t Got it. I've added this task:");
+                int pos = response.indexOf('/');
+                list.add(amount, "[D][ ] "
+                        + response.substring(9, pos - 1)
+                        + " (by: " + response.substring(response.indexOf('/') + 4) + ")");
+                System.out.println("\t   " + list.get(amount));
+                System.out.println("\t Now you have " + (amount + 1) + " tasks in the list.");
+                System.out.println(lineBreak);
+                amount++;
+                save(list);
+            } else if (response.matches("event .+")) {
+                System.out.println(lineBreak);
+                System.out.println("\t Got it. I've added this task:");
+                int pos = response.indexOf('/');
+                list.add(amount, "[E][ ] "
+                        + response.substring(6, pos - 1)
+                        + " (at: " + response.substring(response.indexOf('/') + 4) + ")");
+                System.out.println("\t   " + list.get(amount));
+                System.out.println("\t Now you have " + (amount + 1) + " tasks in the list.");
+                System.out.println(lineBreak);
+                amount++;
+                save(list);
+            } else if (response.equals("todo")) {
+                System.out.println(lineBreak);
+                System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                System.out.println(lineBreak);
+            } else if (response.equals("deadline")) {
+                System.out.println(lineBreak);
+                System.out.println("☹ OOPS!!! The description of a deadline cannot be empty.");
+                System.out.println(lineBreak);
+            } else if (response.equals("event")) {
+                System.out.println(lineBreak);
+                System.out.println("☹ OOPS!!! The description of an event cannot be empty.");
+                System.out.println(lineBreak);
+            } else {
+                System.out.println(lineBreak);

```

Comment:
You could consider abstracting the
```java
System.out.println(lineBreak);
System.out.println(something);
System.out.println(lineBreak);
```
to avoid repeated code